osbs-box.py

```diff
@@ -94,11 +96,24 @@ def up(args):
            "add-cluster-role-to-user", "cluster-admin", "osbs"]
     _run(cmd)
 
+    # Copy distro-specific Dockerfile
+    filename = "Dockerfile.{}".format(args.distro)
+    for directory in ['client', 'hub', 'koji-builder', 'shared-data']:
+        src_filepath = os.path.abspath('/'.join([directory, filename]))
+        destfilepath = os.path.abspath('/'.join([directory, 'Dockerfile']))
+        cmd = ["cp", "-rvf", src_filepath, destfilepath]
+        _run(cmd)
+
     # Build containers
     cmd = ["docker-compose", "build"]
     if args.force_rebuild:
         cmd += ["--no-cache"]
-    _run(cmd)
+    if args.repo_url:
```

Comment:
It's required if you request rhel7 distro but underlying host is not rhel7. In this case, there won't be any available repos to install packages from.